modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIMANAGER5326CustomStatusMsgTestCase.java

```diff
@@ -102,45 +92,39 @@ public void testAPIErrorResponse() throws Exception {
             apiRequest.setTier(APIMIntegrationConstants.API_TIER.UNLIMITED);
 
             //Add the API using the API publisher.
-            response = apiPublisher.addAPI(apiRequest);
-            verifyResponse(response);
+            HttpResponse createResponse = restAPIPublisher.addAPI(apiRequest);
```

Comment:
Let's extend the APIManagerLifecycleBaseTest class and use the common methods

Reply:
Fixed with 69618a8